server/react-app.d.ts

```diff
@@ -1,7 +1,3 @@
-/// <reference types="node" />
```

Comment:
Not sure if these should change to import Types from calls

Reply:
Did these cause issues, or why did you remove them?

Reply:
https://typescript-eslint.io/rules/triple-slash-reference
I thought about adding `import types ...` but since the compiler complained I just removed them. That might be because the compiler isn't running in strict mode right now

Reply:
I think it's fine to keep these removed without adding import statements; `node` is implicit, we're importing React down in the `"*.svg"` module, and it doesn't looks like we're using `react-dom` at all here.